lib/ae_mdw_web/helpers/aexn_helper.ex

```diff
@@ -25,7 +25,9 @@ defmodule AeMdwWeb.Helpers.AexnHelper do
   @spec enc_id(pubkey()) :: String.t()
   def enc_id(pk), do: :aeser_api_encoder.encode(:account_pubkey, pk)
 
-  @spec sort_field_truncate(String.t()) :: String.t()
+  @spec sort_field_truncate(String.t() | :atom) :: String.t()
```

Comment:
this should be `atom()` since `:atom` represents that literal atom

rest looks good 👍

Reply:
true, atom atom (: